pom.xml

```diff
@@ -401,6 +412,17 @@
       <groupId>net.spy</groupId>
       <artifactId>spymemcached</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.arrow</groupId>
+      <artifactId>arrow-memory-core</artifactId>
+      <exclusions>
+        <!-- conflicts with guava's dependency, which is an older version of the checker framework -->
+        <exclusion>
+          <groupId>org.checkerframework</groupId>
+          <artifactId>checker-qual</artifactId>
+        </exclusion>
```

Comment:
I'm not totally certain this is the best way to approach this, but it's a good point for discussion

Reply:
This is marked outdated because I moved the exclusion to the `dependencyManagement` section, but it's still relevant to discuss.